claims.go

```diff
@@ -11,14 +11,30 @@ import (
 // If this string is a valid JSONPointer, it will be interpreted as such to locate
 // the claim. Otherwise, the claim string will be used directly.
 func getClaim(allClaims map[string]interface{}, claim string) interface{} {
+	var val interface{}
+	var err error
+
 	if !strings.HasPrefix(claim, "/") {
-		return allClaims[claim]
+		val = allClaims[claim]
+	} else {
+		val, err = pointerstructure.Get(allClaims, claim)
+		if err != nil {
+			return nil
```

Comment:
Should there be a logged Warning here showing the error?

Reply:
Yeah this function should probably be upgraded to return an error. But IIRC, I couldn't even find a way to make pointerstructure.Get() return an error other than "not found". The compilation errors all came from not starting with '/' which is special-cased before this is even reached. I'll double check though.